src/Dfe.PlanTech.Application/Response/Commands/ProcessCheckAnswerDtoCommand.cs

```diff
@@ -55,10 +55,11 @@ private async Task<CheckAnswerDto> _RemoveDetachedQuestions(List<QuestionWithAns
 
             while (node != null)
             {
-                if (questionWithAnswerMap.ContainsKey(node.Sys.Id))
+                QuestionWithAnswer? questionWithAnswer;
+                if (questionWithAnswerMap.TryGetValue(node.Sys.Id, out questionWithAnswer))
```

Comment:
IMO merge these two lines for readability, i.e.

`if (questionWithAnswerMap.TryGetValue(node.Sys.Id, out QuestionWithAnswer? questionWithAnswer))`

Reply:
Thought something felt off about how I was using that..

Resolved in: [a9356db](https://github.com/DFE-Digital/plan-technology-for-your-school/pull/288/commits/a9356dbe1ffcd87e0779e79484d2610c67c29357)